docs/console/usage-billing.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: usage-billing
+title: Usage-based billing
+sidebar_label: Usage-based billing
+slug: usage-billing
+---
+
+:::info
+
+This feature is in preview. While we are rolling out usage-based billing, we will not collect payment for overusage. Payment
+collection will be rolled out gradually in Q3 2024.
+
+Please report any issues to support@ory.sh or contact us in the [Ory Community](https://slack.ory.sh/).
+
+:::
+
+## Overview
+
+Usage-based invoices are a feature in the Ory Console that allows you to understand your usage of Ory Network services, the usage
+that is included in your plan, and the overuse. Usage-based invoices are generated every month. For the current billing period, a
+draft invoice is generated every six hours.
+
+Usage-based invoices only apply to workspaces and projects contained therein, but not to legacy plans that have individual
+subscriptions.
```

Comment:
Do we have docs for workspaces, and what constitutes a legacy plan already? I imagine this could be very confusing for some people. If so, would be great to link here, otherwise we should write some soon.

It's not in scope for this PR if we don't, though.